library/system.lisp

```diff
@@ -46,21 +48,6 @@
       (trivial-garbage:gc :full cl:t)
       Unit))
 
-  (declare time ((Unit -> :a) -> (Tuple :a Integer)))
-  (define (time f)
-    "Run the thunk `f` and return a tuple containing its value along with the run time in microseconds.
-
-While the result will always contain microseconds, some implementations may return a value rounded to less precision (e.g., rounded to the nearest second or millisecond)."
-    (let start = (lisp Integer () (cl:get-internal-run-time)))
-    (let value = (f))
-    (let end   = (lisp Integer () (cl:get-internal-run-time)))
-    (Tuple value
-           (lisp Integer (start end)
-             (cl:values
-              (cl:round
-               (cl:* 1000000 (cl:- end start))
-               cl:internal-time-units-per-second)))))
-
   (declare sleep (Integer -> Unit))
   (define (sleep n)
```

Comment:
Maybe file an issue about this. CL:SLEEP can take any non-negative real as an argument.

Reply:
[filed](https://github.com/coalton-lang/coalton/issues/1304)